### PR TITLE
Fix missing description and http --> https

### PR DESCRIPTION
**Pull request by velkovb**

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/dhcpserver/Get-DhcpServerv6FreeIPAddress.md [hunk 1]
```diff
@@ -83,8 +83,8 @@ Use this parameter to run commands that take a long time to complete.
 The cmdlet immediately returns an object that represents the job and then displays the command prompt.
 You can continue to work in the session while the job completes.
 To manage the job, use the `*-Job` cmdlets.
-To get the job results, use the [Receive-Job](http://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
-For more information about Windows PowerShell® background jobs, see [about_Jobs](http://go.microsoft.com/fwlink/?LinkID=113251).
+To get the job results, use the [Receive-Job](https://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
+For more information about Windows PowerShell® background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```
Comment (JohanFreelancer9) [inline docset/windows/dhcpserver/Get-DhcpServerv6FreeIPAddress.md]:
```suggestion
For more information about Windows PowerShell background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```

File: docset/windows/dhcpserver/Get-DhcpServerv6FreeIPAddress.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ ms.assetid: 0A64F272-2D6C-437E-8788-B7C38065F390
 # Get-DhcpServerv6FreeIPAddress
 
 ## SYNOPSIS
-Gets free IPv6 addresses from a scope.
+Gets one or more free, or unassigned, IPv6 addresses from the specified scope.
```
Comment (JohanFreelancer9) [inline docset/windows/dhcpserver/Get-DhcpServerv6FreeIPAddress.md]:
```suggestion
Gets one or more free or unassigned IPv6 addresses from the specified scope.
```

File: docset/winserver2012-ps/dhcpserver/Get-DhcpServerv6FreeIPAddress.md [hunk 3]
```diff
@@ -25,15 +25,15 @@ Get-DhcpServerv6FreeIPAddress [-Prefix] <IPAddress> [[-NumAddress] <UInt32>] [-A
 ## DESCRIPTION
 The **Get-DhcpServerv6FreeIPAddress** cmdlet gets one or more free IPv6 addresses from the specified scope.
 
-If the **NumAddress** parameter is specified, then the requested number of free IPv6 addresses are returned.
-If the **NumAddress** parameter is not specified, then a single free IPv6 address will be returned.
+If the *NumAddress* parameter is specified, then the requested number of free IPv6 addresses are returned.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/dhcpserver/Get-DhcpServerv6FreeIPAddress.md]:
```suggestion
If the *NumAddress* parameter is specified, then the requested number of free IPv6 addresses is returned.
```

File: docset/winserver2012-ps/dhcpserver/Get-DhcpServerv6FreeIPAddress.md [hunk 4]
```diff
@@ -69,7 +69,13 @@ This example gets a free IPv6 address from the specified scope within the IP add
...
 
 ### -AsJob
-ps_cimcommon_asjob
+Runs the cmdlet as a background job.
+Use this parameter to run commands that take a long time to complete. 
+The cmdlet immediately returns an object that represents the job and then displays the command prompt.
+You can continue to work in the session while the job completes.
+To manage the job, use the `*-Job` cmdlets.
+To get the job results, use the [Receive-Job](https://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
+For more information about Windows PowerShell® background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/dhcpserver/Get-DhcpServerv6FreeIPAddress.md]:
```suggestion
For more information about Windows PowerShell background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```

File: docset/winserver2012r2-ps/dhcpserver/Get-DhcpServerv6FreeIPAddress.md [hunk 5]
```diff
@@ -33,15 +33,15 @@ Get-DhcpServerv6FreeIPAddress [-ComputerName <String>] [-Prefix] <IPAddress> [[-
 ## DESCRIPTION
 The **Get-DhcpServerv6FreeIPAddress** cmdlet gets one or more free IPv6 addresses from the specified scope.
 
-If the **NumAddress** parameter is specified, then the requested number of free IPv6 addresses are returned.
-If the **NumAddress** parameter is not specified, then a single free IPv6 address will be returned.
+If the *NumAddress* parameter is specified, then the requested number of free IPv6 addresses are returned.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/dhcpserver/Get-DhcpServerv6FreeIPAddress.md]:
```suggestion
If the *NumAddress* parameter is specified, then the requested number of free IPv6 addresses is returned.
```

**Pull request merged**